src/rmp/src/delay_optimization_strategy.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include "delay_optimization_strategy.h"
+
+#include "abc_library_factory.h"
+#include "base/abc/abc.h"
+#include "map/mio/mio.h"
+#include "map/scl/sclLib.h"
+#include "map/scl/sclSize.h"
+#include "utl/deleter.h"
+
+namespace abc {
+extern Abc_Ntk_t* Abc_NtkMap(Abc_Ntk_t* pNtk,
```

Comment:
Is this not defined in an abc header than you can include?

Reply:
No it's not, I can send a PR to ABC to fix that, but would rather not block this PR.

Reply:
I won't block it but it would be nice to fix this and the const issue in abc.